### PR TITLE
SPUI-953 > Fix bottom button and fix share button in invite friends bo…

### PR DESCRIPTION
**Pull request by DenisMartych**
…ttomsheet

### CODE REVIEW

File: plugins/simple_kit/lib/src/bottom_sheets/components/basic_bottom_sheet/components/basic_bottom_sheet.dart [hunk 1]
```diff
@@ -176,5 +181,9 @@ double _listViewMaxHeight({
     }
   }
 
+  if (pinnedBottom != null) {
+    max = max - 104;
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/bottom_sheets/components/basic_bottom_sheet/components/basic_bottom_sheet.dart:185]: make size reactive

File: plugins/simple_kit/lib/src/referral_invite/simple_referral_invite_bottom_pinned.dart [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          ),
+        ),
+      ),
+      child: SPaddingH24(
+        child: SPrimaryButton2(
+          onTap: onShare,
+          name: 'Share',
+          active: true,
+          icon: const SShareIcon(
+            color: Colors.white,
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/referral_invite/simple_referral_invite_bottom_pinned.dart:33]: use SColorsLight().white

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: plugins/simple_kit/lib/src/bottom_sheets/components/basic_bottom_sheet/components/basic_bottom_sheet.dart (hunk 1 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/bottom_sheets/components/basic_bottom_sheet/components/basic_bottom_sheet.dart:185]: done

File: plugins/simple_kit/lib/src/referral_invite/simple_referral_invite_bottom_pinned.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/referral_invite/simple_referral_invite_bottom_pinned.dart]: done

**Pull request merged**